62.CAD/common.hlsl

```diff
@@ -1,4 +1,6 @@
-
+#ifndef __cplusplus
+#include <nbl/builtin/hlsl/shapes/beziers.hlsl>
+#endif
```

Comment:
Can you look into the compile errors we get when including this in cpp?
Cause if we can do that, we can move some of the stuff there such as `QuadBezierAnalyticArcLengthCalculator` or merge that with Precomp struct

Reply:
![image](https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/assets/32484732/ee11b211-a176-4193-bdfb-53474fdf36a1)

Reply:
Ok so it's not that much. can you fix these? tbh I don't know what these errors mean mostly unless I see the code besides them